package-lock.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@okta/okta-sdk-nodejs",
```

Comment:
This project uses `yarn` and should not have a `packge-lock.json` file (this is for `npm`). This file should be removed